server/src/routes/event.ts

```diff
@@ -1,9 +1,62 @@
 import express from "express";
-
+import { report } from "@/utils/error";
+import context from "@/context";
 const router = express.Router();
 
-router.post("/create-single-event", (req, res) => {
-  // ...
+router.post("/add-single-event", async (req, res) => {
```

Comment:
1. `npm i body-parser && npm i -D @types/body-parser`
2. Add `body-parser` to this post request
```typescript
import bodyParser from "body-parser"
// other imports

// ...

router.post("/add-single-event", bodyParser.json(), async (req, res) => {
  // ...
});
```